ui/tests/unit/services/permissions-test.js

```diff
@@ -0,0 +1,137 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import Pretender from 'pretender';
+
+const PERMISSIONS_RESPONSE = {
+  data: {
+    exact_paths: {
+      foo: {
+        capabilities: ['read'],
+      },
+      'bar/bee': {
+        capabilities: ['create'],
+      },
+      boo: {
+        capabilities: ['deny'],
+      },
+    },
+    glob_paths: {
+      'baz/biz': {
+        capabilities: ['read'],
+      },
+    },
+  },
+};
+
+module('Unit | Service | permissions', function(hooks) {
+  setupTest(hooks);
+
+  hooks.beforeEach(function() {
+    this.server = new Pretender();
+    this.server.get('/v1/sys/internal/ui/resultant-acl', () => {
+      return [200, { 'Content-Type': 'application/json' }, JSON.stringify(PERMISSIONS_RESPONSE)];
+    });
+  });
+
+  hooks.afterEach(function() {
+    this.server.shutdown();
+  });
+
+  test('sets paths properly', async function(assert) {
+    let service = this.owner.lookup('service:permissions');
+    await service.getPaths.perform();
+    assert.deepEqual(service.get('exactPaths'), PERMISSIONS_RESPONSE.data.exact_paths);
+    assert.deepEqual(service.get('globPaths'), PERMISSIONS_RESPONSE.data.glob_paths);
+  });
```

Comment:
Can you see what would happen if I wanted to skip this ^ test temporarily, or I was in the QUnit browser runner and I clicked to run only one of the below tests again? I'm guessing here, but I'm pretty sure it would be best to tweak this slightly.

Reply:
i tried skipping the test and rerunning it several times and all of the tests still pass. hurray!

Reply:
Oh wow! Hehe, that'll teach me to not try things out! 😂 

Ok so that leads onto my next question - why is the call to `await service.getPaths.perform();` needed here yet not in the other tests? Ah just looked properly and answered my own questions, `set` is not `get`! In my head I thought you were just getting and comparing stuff there, not setting and then getting and comparing!

Sorry completely misread this the other day! Thanks for looking anyway!